src/api/index.js

```diff
@@ -1,5 +1,13 @@
-import { USER_URL, USER_LOGIN_URL, METHODS ,UPDATE_DATA} from "./constants";
+import {
+  USER_URL,
+  USER_LOGIN_URL,
+  METHODS,
+  UPDATE_DATA,
+  ABOUT_USER
+} from "./constants";
 import GlobalAPISvc from "./globalApi";
+import * as auth from "../services/Session";
+const uuid = auth.getItem("uuid");
 
 const postUserData = (data, resolve, reject) => {
```

Comment:
what is this naming convention?

Reply:
are you talking about `auth` or `uuid`?

Reply:
I'm talking about postUserData

Reply:
I have renamed `uuid` to `userId`.
auth = > authentication

Reply:
renamed it